content/soarian/dstu2.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Soarian 
```

Comment:
fixed. @slvsridhar please confirm that the Soarian server supports the _format parameter or not (if not, I'll need to remove it from the media types section)